spectator-api/src/main/java/com/netflix/spectator/impl/Scheduler.java

```diff
@@ -326,17 +337,28 @@ void updateNextExecutionTime(Counter skipped) {
      * @param skipped
      *     Counter that will be incremented each time an expected execution is
      *     skipped when using {@link Policy#FIXED_RATE_SKIP_IF_LONG}.
+     * @param lastSuccessfulExecutionTimestamp
+     *     AtomicLong that will be updated to the current wallclock time of a successful run. This is useful for
+     *     monitoring.
+     * @param lastFailedExecutionTimestamp
+     *     AtomicLong that will be updated to the current wallclock time of a failed run. This is useful for
+     *     monitoring.
      */
-    void runAndReschedule(DelayQueue<DelayedTask> queue, Counter skipped) {
+    void runAndReschedule(DelayQueue<DelayedTask> queue,
```

Comment:
This should align with the margin not the opening paren. Aligning with opening braces causes future diffs to have a lot of noise just to preserve the alignment.